### PR TITLE
Licenses functionality

### PR DESCRIPTION
**Pull request by Lkubok**
# Tasks:
- [ ] for ex. document templates management

# Added value:
- [ ] for ex. Users can easier/faster create new documents based on a predefined templates

# JIRA references:

# PR dependency:
[1] for ex. http://github.com/PullRequest_url

# Testing scenario:
1. Go there
2. Do that
3. etc..

# Preview
Paste some screenshot to simplify review
[IMAGE ALT TEXT HERE](https://www.youtube.com/watch?v=YOUTUBE_VIDEO_ID_HERE)

```diff
- Don't forget about updating JIRA task with PR #ID, added value information and Testing scenarios
```

### CODE REVIEW

File: src/commands/check.ts [hunk 1]
```diff
@@ -19,6 +19,11 @@ export default class Check extends Command {
         name: 'Default Issue Template',
         status: false,
         callback: checkDefaultIssueTemplate
+      },
+      {
+        name: 'Default Issue Template',
```
Comment (PoslinskiNet) [inline src/commands/check.ts:24]: @Lkubok I think its not the `Issue`.

File: src/commands/license.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import { Command } from '@oclif/command'
+import { checkLicense } from '../common'
```
Comment (PoslinskiNet) [inline src/commands/license.ts:2]: Do we have it here? I can't see it in this PR.

Comment (Lkubok) [reply src/commands/license.ts:2]: in common.js we have import like import { findExistingDefaultLicenseTemplate, checkLicense } from './features/license'

and next it is exported, so we have it.

File: src/commands/check.ts (hunk 1 above)
Comment (Lkubok) [reply src/commands/check.ts:24]: done

File: src/features/license/checkLicense.ts [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+      .prompt({
+        name: 'template-issue-default',
+        type: 'list',
+        message: 'License found, Would You like to change License?',
+        choices: ['Yes', 'No'],
+        default: 'Yes',
+        suffix: '\n(ctrl + c to exit)'
+      })
+      .then(async answer => {
+        if (answer['template-issue-default'] === 'Yes') {
```
Comment (PoslinskiNet) [inline src/features/license/checkLicense.ts:28]: template-issue-default? @Lkubok

**Pull request merged**